utils/material_plots.py

```diff
@@ -30,16 +32,14 @@ def main():
         for i in range(nMat):
             material = entry.material.at(i)
 
-            # If you need to replace some string in the material, add that here
-            material = material.replace("66D","")
-            material = material.replace("Vtx","")
+            # Removing substrings from materials
+            for substring in args.removeMatsSubstrings:
+                material = material.replace(substring,"")
```

Comment:
Isn't there an indentation problem here?

Reply:
I don't see a problem, the indentation is the same as everywhere else (four spaces) and the "Removing substrings from materials" and "Ignore certain materials if specified" functionalities should be at the same level

Reply:
Indeed, I mis-read, sorry